### PR TITLE
appending cmd tab in trying out bind mounts section to work Closes #18242

### LINKED ISSUE #18242: 06_bind_mounts command prompt not working
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Other

### Description

Hi,
In section of "trying-out-bind-mounts" there was not a command for command prompt(cmd),  just the powershell is given,  the command that is given it dont work in cmd , i have fixed it and make new tab for command prompt to work in cmd.

### Location

https://docs.docker.com/get-started/06_bind_mounts/

### Suggestion

_No response_

### PR DESCRIPTION
**Pull request by arminshfatemi**

### CODE REVIEW

File: content/get-started/06_bind_mounts.md [hunk 1]
```diff
@@ -59,22 +59,31 @@ work.
    ```
    
    {{< /tab >}}
-   {{< tab name="Windows" >}}
+   {{< tab name="Windows-Powershell" >}}
 
    Run this command in PowerShell.
```
Comment (craig-osterhout) [inline content/get-started/06_bind_mounts.md:64]: Can probably remove this line since it becomes redundant with the tab.
```suggestion

```

**Review by craig-osterhout — CHANGES_REQUESTED** [review]
Thanks for adding the command.
Commented on a few style nits and a numbering issue.

**Review by arminshfatemi — COMMENTED** [review]
looks like this line was ok

**Review by arminshfatemi — COMMENTED** [review]
looks like dont need to change

**Review by arminshfatemi — COMMENTED** [review]
looks like it didnt need to change

File: content/get-started/06_bind_mounts.md [hunk 2]
```diff
@@ -59,14 +59,21 @@ work.
...
+   {{< tab name="Windows (PowerShell)" >}}
 
-   Run this command in PowerShell.
 
    ```powershell
    $ docker run -it --mount "type=bind,src=$pwd,target=/src" ubuntu bash
    ```
    
+   {{< /tab >}}
+   {{< tab name="Windows-CMD" >}}
```
Comment (craig-osterhout) [inline content/get-started/06_bind_mounts.md:70]:
```suggestion
   {{< tab name="Windows (CMD)" >}}
```

**Review by craig-osterhout — APPROVED** [review]
Thank you. LGTM

**Pull request merged**